apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -52,8 +52,30 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
 
         raw: RAW_DATA_FIELD,
 
+        /**
+         * A note on toPay:
+         *
+         * We store all payment data as a string, with (.) as delimiter:
+         * We store currency in BillingIntegration in this format. This information helps client to distinguish the partial part:
+         *
+         * refs: https://gist.github.com/Fluidbyte/2973986
+         *
+         * "RUB": {
+         *		"symbol": "RUB",
+         *		"name": "Russian Ruble",
+         *		"symbol_native": "₽.",
+         *		"decimal_digits": 2,
+         * 		"rounding": 0,
+         *		"code": "RUB",
+         *		"name_plural": "Russian rubles"
+         *	},
+         *
+         * 123.30 (for 123 Ruble and 30 Kopeck)
+         * 6.20 (for 6$ and 20 Cents)
+         *
+         */
         toPay: {
```

Comment:
Offtop: why `toPay` was choosen as name and not `amount`, for example.
Usually, in receipts from internet platforms, I facing following terms: `amount`, `total`, `subTotal`, `grandTotal`…
Take a look at Amazon order details, for example:
<img width="287" alt="image" src="https://user-images.githubusercontent.com/2064830/130812912-4ca30bf6-cfe0-47a7-af61-b9e67facf3ef.png">
IMHO, it's clear, that amount of Receipt is to be paid, isn't? ;)

Reply:
:(

Reply:
I think it was chosen because some billings use somewhat similar. Mabye `amount` would look better on this, but this is matter of another task

http://open-gkh.ru/Bills/exportPaymentDocumentResultType/PaymentDocument.html